core/driver/sources/src/main/resources/reference.conf

```diff
@@ -161,34 +201,108 @@ probe {
+        // throw an exception and fail the test. In other words, here you can define, which IDE errors
+        // should be ignored even if `driver.check.errors.enabled` is set to `true`.
+        //
+        // Both for `includeMessages` and for `excludeMessages` you can define an entry in one of two possible ways:
+        // 1. By using a regex-based string. For example: ".*at com.intellij.uast.UastMetaLanguage.*" will match
+        //    any error message that contains "at com.intellij.uast.UastMetaLanguage". Note: your regex-based string
+        //    CAN NOT be a multi-line string.
+        // 2. By using a raw string without regular expressions. In such case ide-probe just checks if IDE error message
+        //    contains your raw string. The benefit of this option is that you can use a multi-line string (part of
+        //    stacktrace). You don't have to worry about line indentations here as each line is trimmed before performing
+        //    the check. If you want to use a multi-line stirng, use triple quotes: """<your multi-line string...>""".
+        //
+        // If an IDE error matches both an entry from `includeMessages` and another entry from `excludeMessages`,
+        // then such IDE error will be ignored (`excludeMessages` takes precedence).
         includeMessages = [".*"]
         excludeMessages = []
       }
+
+      // `driver.check.freezes.enabled` defines if ide-probe should throw an exception and fail the test if IDE freezes
+      // have occured during test. By default, this functionality is disabled. Set to `true` to enable.
       freezes.enabled = false
     }
+
+    // `driver.display` defines the display mode which should be used for the IDE run. By default, it is "native",
+    // which means that everything is displayed on the screen and you can interact with the IDE during test.
+    // Another possible value is "xvfb" which makes the IDE run in the xvfb mode (no display). This config can be set
+    // either with a raw string value or by setting the IDEPROBE_DISPLAY environment variable which overrides the default value.
     display = "native" # OR "xvfb" - these are two possible values
     display = ${?IDEPROBE_DISPLAY}
+
+    // `driver.xvfb` contains parameters that are used if IDE runs in the xvfb mode. Basically, it contains three
+    // screen size parameters that are passed to the "−screen" command line switch of xvfb.
+    // See https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml for more details.
     xvfb {
       screen {
         width = 1920
         height = 1080
         depth = 24
       }
     }
+
+    // `driver.headless` defines if IDE should run in a headless mode (without a graphical user interface). If you want
+    // to use it, set to `true`. Specifically, it adds " headless" to the basic launch command ("idea").
+    // Note: if `headless` is set to `true`, it takes precedence over `driver.display`, which means that
```

Comment:
Yes, these are two different ways of running IDE. In fact, it seems to me that we could fix the way ide-probe is configured so that you can choose in only one config, which way should be chosen:
a) native
b) xvfb
c) headless mode
<img width="1058" alt="image" src="https://user-images.githubusercontent.com/49597713/197882059-cf457b44-fc7d-46c8-8088-a47d6e94148e.png">

Reply:
+1, makes sense